<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>801</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 60, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Wawati SC</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(195, 255, 89);</string>
     </property>
     <property name="text">
      <string>工作软件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#menu_1{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_1:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_1:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#menu_2{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_2:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_2:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#menu_3{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_3:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 
QPushButton#menu_3:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>120</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget#widget_1{
background-color: rgba(200, 200, 200);
border:2px solid white;border-radius:10px;
}
</string>
    </property>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>120</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget#widget_2{
background-color: rgba(200, 200, 200);
border:2px solid white;border-radius:10px;
}
</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="model_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
